src/components/MoneyRequestConfirmationList.js

```diff
@@ -494,6 +520,16 @@ function MoneyRequestConfirmationList(props) {
                             disabled={didConfirm || props.isReadOnly}
                         />
                     )}
+                    {canUseTags && tagList ? (
```

Comment:
I think if you use AND here then you don't need the `: null`

```suggestion
                    {canUseTags && tagList && (
```